롬복 의존성 미설정으로 인한 생성자 오류 발생

## 💡 Issue
롬복 의존성 설정 미완료로 인해 생성자 오류 발생

---

### ✏️ Description
```
/Users/dogiheon/Desktop/Team5_BE/src/main/java/ojosama/talkak/common/exception/code/VideoError.java:10: error: constructor VideoError in enum VideoError cannot be applied to given types;
    VIDEO_NOT_FOUND(HttpStatus.BAD_REQUEST, "V001", "존재하지 않는 영상입니다.");
                   ^
  required: no arguments
  found:    HttpStatus,String,String
  reason: actual and formal argument lists differ in length
```

---

### ✅ To-do List

---
- [x] 롬복 의존성 세팅 마무리

--- a/build.gradle
+++ b/build.gradle
@@ -23,12 +23,12 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-webflux'
 	implementation 'org.projectlombok:lombok'
-    annotationProcessor 'org.projectlombok:lombok'
+	annotationProcessor 'org.projectlombok:lombok'
 	runtimeOnly 'com.h2database:h2'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
-    testImplementation 'org.projectlombok:lombok'
-    testAnnotationProcessor 'org.projectlombok:lombok'
+	testImplementation 'org.projectlombok:lombok'
+	testAnnotationProcessor 'org.projectlombok:lombok'
 }
 
 tasks.named('test') {

--- a/build.gradle
+++ b/build.gradle
@@ -23,9 +23,12 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-webflux'
 	implementation 'org.projectlombok:lombok'
+    annotationProcessor 'org.projectlombok:lombok'
 	runtimeOnly 'com.h2database:h2'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
+    testImplementation 'org.projectlombok:lombok'
+    testAnnotationProcessor 'org.projectlombok:lombok'
 }
 
 tasks.named('test') {

